SharedProjects/Utilities/TextureUtilities.cs

```diff
@@ -0,0 +1,311 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.Drawing.Imaging;
+using System.IO;
+using BabylonExport.Entities;
+using GLTFExport.Entities;
+
+namespace Utilities
+{
+    class TextureUtilities

```

Comment:
could the class be static as well ? reenforcing the intent of utils contained in its name ?

Reply:
good point, I'll add that as well.